### PR TITLE
Update to latest libocr and enable bootstrap node telemetry

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

File: core/services/validators.go [hunk 1]
```diff
@@ -438,6 +438,9 @@ func ValidatedOracleSpecToml(tomlString string) (spec offchainreporting.OracleSp
 	if err := validateTimingParameters(spec); err != nil {
 		return spec, err
 	}
+	if err := validateMonitoringURL(spec); err != nil {
```
Comment (j16r) [inline core/services/validators.go:441]: Why is the monitoring URL in the spec? Should be in the global config right? There is only ever one config at a time.

File: core/internal/cltest/cltest.go [hunk 2]
```diff
@@ -1481,3 +1482,22 @@ func RandomizeNonce(t *testing.T, s *strpkg.Store) {
 	err := s.DB.Exec(`UPDATE keys SET next_nonce = ?`, n).Error
 	require.NoError(t, err)
 }
+
+func NewConfigDigest(t *testing.T) ocrtypes.ConfigDigest {
```
Comment (kaleofduty) [inline core/internal/cltest/cltest.go:1486]: Since it does not return a pointer, would `MakeConfigDigest` be a more idiomatic name?

**Review by kaleofduty — COMMENTED** [review]
@connorwstein I believe you added some jobspec validation code that currently enforces that a monitoringEndpoint can only be set for non-bootstrap nodes. 
Since a monitoringEndpoint can now also be set for bootstrap nodes, we will probably have to change that, but I don't know where the validation code is located. Could you point us to it?

**Comment by connorwstein** [comment]
@kaleofduty the code is here https://github.com/smartcontractkit/chainlink/blob/1184476f72e21382ade7c36f6a6684b34dcaaf08/core/services/validators.go#L444 but we already updated it to make monitoring optional for both

**Comment by kaleofduty** [comment]
> but we already updated it to make monitoring optional for both

I see, thank you. Sorry for the spam in that case!

File: core/services/validators.go (hunk 1 above)
Comment (samsondav) [reply core/services/validators.go:441]: I thought we did this deliberately to enable node operators to set multiple monitoring endpoints... Is that a mistake?

Comment (samsondav) [reply core/services/validators.go:441]: Yup, so we want a per-job configuration option but the default is to fall back to the global one. Check latest changes

**Review by kaleofduty — APPROVED** [review]

File: core/services/validators.go (hunk 1 above)
Comment (j16r) [reply core/services/validators.go:441]: Seems a little YAGNI to me and is going to require a rearchitect of how it all works behind here.

**Pull request merged**